contracts/hardhat.config.ts

```diff
@@ -71,6 +71,13 @@ let config = {
         version: '0.8.19',
         settings: COMPILER_SETTINGS,
       },
+      {
+        version: '0.8.20',
```

Comment:
Why is this set to 0.8.20 but the compile script to 0.8.20?

Reply:
Update the reference that was forcing us to use this version and removed this version